modules/objdetect/include/opencv2/objdetect/face.hpp

```diff
@@ -75,7 +75,7 @@ class CV_EXPORTS_W FaceDetectorYN
     CV_WRAP static Ptr<FaceDetectorYN> create(const String& model,
                                               const String& config,
                                               const Size& input_size,
-                                              float score_threshold = 0.9f,
+                                              float score_threshold = 0.5f,
```

Comment:
It is a part of public API.
Changing default values between releases should be avoided as users already expect reliable (0.9) results here.

Reply:
Hello, I have restored the original defaults, thanks.